README.md

```diff
@@ -47,10 +46,37 @@ The methods of the `PSP22Data` class correspond directly to queries and operatio
  - Add definitions of `Transfer` and `Approval` events in the body of your contract.
  - Add the `impl PSP22 for [struct_name]` block with implementation of PSP22 trait messages using `PSP22Data` methods. Each method which mutates the state of the token database returns a `Result<Vec<PSP22Event>, PSP22Error>` with all events generated by that operation. Please make sure to handle errors correctly and emit the resulting events (see the `emit_events` function).
  - Optionally implement also the `PSP22Metadata` trait to make your token play nice with other ecosystem tools.
+ - Optionally add unit tests with `tests!` macro (see below)
 
 The contract in [`lib.rs`][lib] contains an example implementation following all the above steps. Feel free to copy-paste parts of it.
 
-### 4. Burnable and Mintable extensions
+### 4. Unit testing
+
+This crate comes with a suite of unit tests for PSP22 tokens. It can be easily added to your contract's unit tests with a helper macro `tests!`. The macro should be invoked inside the main contract's module (the one annotated with `#[ink::contract]`):
```

Comment:
The default contract template (flipper) does put tests inside contract module...

The reason is mainly to simplify the imports inside the macro code.

Reply:
If I'm not mistaken, if the test macro was invoked outside of contract's module, it would need to know the name of that module (to import things like events from inside of it). And that would probably require one more macro argument, which is something I'd like to avoid (the requirement for constructor is already kinda painful :/)

Reply:
Yeah, from what I've noticed it is sort of a common place to put the tests in. It does ease imports a lot